db/schema.rb

```diff
@@ -11,11 +11,32 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170123205547) do
+ActiveRecord::Schema.define(version: 20170223174042) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "episode_images", force: :cascade do |t|
```

Comment:
So in this new world, we've got `itunes_images`, `episode_images` and `podcast_images`.  The latter 2 make sense... but what is `itunes_images` used for, going forward?  (I thought maybe the "type" on the new image tables would be used to differentiate between the itunes/non-itunes channel image).